swagger.yaml

```diff
@@ -91,6 +96,122 @@ paths:
           $ref: "#/responses/InternalError"
         502:
           $ref: '#/responses/BadGateway'
+  /population-types/{population-type}/dimensions:
+    get:
+      summary: "Get list of non-geography base variables for given dataset "
+      description: ""
+      produces:
+        - "application/json"
+      parameters:
+        - name: population-type
+          type: string
+          in: path
+          required: true
+      responses:
+        200:
+          description: "A json list containing non-geography base variables for a dataset"
+          schema:
+            $ref: '#/definitions/Dimension'
```

Comment:
Isn't this returning a `GetDimensionsResponse`?

Reply:
Now updated